src/client.py

```diff
@@ -16,33 +25,31 @@ def client(message):
     message += '\r\n'
 
     if version_info[0] == 2:
```

Comment:
This version check is not the best way to handle Python 2/3 differences. A better way is to declare `from __future__ import unicode_literals` at the top of your page. This way, you are forcing both versions of Python to use unicode strings, and your `encode` call before sending the bytes data will always work.